tests/lowering/tensor_manipulation/test_unsqueeze.py

```diff
@@ -64,7 +65,8 @@ def test_unsqueeze2(device, input_shape, dim):
 
 @pytest.mark.parametrize(
     "input_shape, dim",
-    [((5, 2, 4, 3), -2)],
+    # [((5, 2, 4, 3), -2)],
```

Comment:
can you leave a comment in the code on why this is commented and not removed and what should be done to uncomment? ideally with a link to github issue

Reply:
A [commit](https://github.com/tenstorrent/pytorch2.0_ttnn/commit/6903996c10c2d6eef35910a943ccf5360c03e825) I made for GPT-2 affected this test, but I reverted it since GPT-2 isn't working entirely.